src/openapi.test.ts

```diff
@@ -133,28 +133,37 @@ describe('toOpenAPISpec', () => {
                 in: 'query',
                 name: 'sort',
                 required: true,
-                schema: { type: 'string' },
+                schema: { enum: ['asc', 'desc'] },
```

Comment:
😅 so we totally don't use `type: 'string'` as default type at all?

Reply:
In this case, it's possible that the "best practice" would be to specify `type: string` in addition to the `enum`. But, I don't think it's essential, and regardless, it's up to the consumer how they define their schemas.

Reply:
sure, I thought `type: 'string'` is a default schema for those who didn't declare it in `schema.(yaml|json)` as a feature, now it becomes totally optional (speaking of up to the consumer).